src/build.sh

```diff
@@ -124,6 +128,41 @@ function build_kernel_main()
     return "$?"
   fi
 
+  if [[ -n "$from_sha_arg" ]]; then
+    # Check if there is a rebase in process.
+    if [[ -d .git/rebase-merge ]]; then
+      printf '%s\n' 'ERROR: Abort the repository rebase before continuing with build from sha (use "git rebase --abort")!'
+      return 22 #EINVAL
```

Comment:
Add a space between `#` and `EINVAL`. Do it for every addition you've introduced like this

Reply:
@MarceloSpessoto